src/sass/patterns/_form-elements.scss

```diff
@@ -1,16 +1,47 @@
 // Form Elements
 // =============
 
-.vertical-separator {
-  border-right: 1px solid $color-border-separator-alt;
+.form-grid {
+  display: grid;
+  grid-template-columns: 1fr 25%;
+  flex-grow: 1;
+  width: calc(100% - 2.25rem);
```

Comment:
The issues here are caused by using flex to achieve grid-like layouts. If you could make the `slds-accordion__summary` act as a grid container, rather than a flex container, you could eliminate both these lines. The only magic number you'd really need is margin/padding on the header row to get it the same width as accordion rows.

Another option is to blame `.slds-has-flexi-truncate` on the `h3` - which is allowing it to grow more than it should. That seems like a more complicated fix to me, though.

Reply:
I don't think I will be able to add a class to `slds-accordion__summary` but I can target our custom `data-form='task-capture'` and then the `slds-accordion__summary`